examples/format-imperative/README.md

```diff
@@ -25,6 +25,4 @@ $ kpt fn eval format-imperative --image gcr.io/kpt-fn/format:unstable
 
 ## Expected result
 
-The fields should be ordered as per OpenAPI schema definition of `Deployment`
-resource. For e.g. `metadata.name` field is placed before `metadata.labels`
-field. The keys in `metadata.labels` field are sorted alphabetically.
+Consistent indentation has been enforced.
```

Comment:
This is not true. The input indentation is preserved.

Reply:
I will update the format docs in a separate PR. For now you can just delete this commit and unblock the CI

Reply:
This is what I observed. Did I miss anything?
Before 
```yaml
apiVersion: apps/v1
kind: Deployment
metadata:
  labels:
     foo: bar # 5 leading whitespaces
     env: dev # 5 leading whitespaces
  name: my-nginx
  namespace: my-space
spec:
  selector:
   matchLabels: # 3 leading whitespaces
      app: nginx
  replicas: 3
  template:
    metadata:
      labels:
        app: nginx
    spec:
      containers:
      - name: nginx  # no additional indentation for list
        image: nginx:1.14.2
        ports:
          - containerPort: 80 # 2 additional whitespaces indentation for list
```

After
```yaml
apiVersion: apps/v1
kind: Deployment
metadata:
  labels:
    foo: bar # 5 -> 4 leading whitespaces
    env: dev # 5 -> 4 leading whitespaces
  name: my-nginx
  namespace: my-space
spec:
  selector:
    matchLabels: # 3 -> 4 leading whitespaces
      app: nginx
  replicas: 3
  template:
    metadata:
      labels:
        app: nginx
    spec:
      containers:
      - name: nginx  # no additional indentation for list
        image: nginx:1.14.2
        ports:
        - containerPort: 80 # no additional indentation for list 
```

Reply:
It retains indentation per resource. If within the same resource, fields have different indentation, kpt uses the first sequence node indentation and makes it consistent per resource. In any case, it is purely related to kpt cli and not related to format function. You can try this with other functions as well and the behavior will be same.

Reply:
And the map node indentation as well `# 5 -> 4 leading whitespaces` is done by kpt cli and not the format function.

Reply:
I have reverted the last 2 commits.
The format example has been skipped.
@phanimarupaka I will let you figure out what to do and update the usage and example doc.